src/parser/jobs/rdm/Gauge.js

```diff
@@ -223,94 +243,69 @@ export default class Gauge extends Module {
 					content: <Fragment>
 						<Message warning icon>
 							<Icon name="warning sign"/>
-								Due to a missing cast at the start of the log, mana calculations might be off.
-								Additionally 1 or more finishers might have been incorrectly flagged as wrongly used.
+							<Trans id="rdm.gauge.missing-cast.warning.content">Due to a missing cast at the start of the log, mana calculations might be off.
```

Comment:
Potentially minor (ackwell can have the final say here), but I think the standard we've been using for string IDs is `[job].[module].[section if applicable].[the rest of it]`. So for all of these ones, you'd want to be using `rdm.gauge.suggestions...` as the start of the key instead of just `rdm.gauge...`.

Reply:
That's mostly me being anal. Helps split between "gauge checklist" and "gauge suggestions" and "gauge output" (say).

I won't block on it, but going to leave this overnight.

Reply:
rdm.gauge.suggestions.missing-cast-warning-content
rdm.gauge.suggetsions.missing-cast-warning-why
etc.

Gonna push this change up, I honestly don't care what they are named one way or another.